05-services.md

```diff
@@ -1025,11 +1012,15 @@ Compose creates containers with canonical labels:
 - `com.docker.compose.project` set on all resources created by Compose to the user project name
 - `com.docker.compose.service` set on service containers with service name as defined in the Compose file
 
-The `com.docker.compose` label prefix is reserved. Specifying labels with this prefix in the Compose file must
-result in a runtime error.
+The `com.docker.compose` label prefix is reserved. Specifying labels with this prefix in the Compose file
+results in a runtime error.
 
 ### links
 
+> **Note**
```

Comment:
@glours would this be enough to clarify? Or shall I still specifically call out the merging of v2andv3 file format?

Reply:
IHMO is good enough, we don't need to details why it was deprecated in Swarm, just let users know some implementations may not support it is enough